service/src/main/java/com/ge/predix/acs/privilege/management/dao/ParentEntity.java

```diff
@@ -0,0 +1,41 @@
+package com.ge.predix.acs.privilege.management.dao;
+
+import java.util.Set;
+
+import com.ge.predix.acs.model.Attribute;
+
+public class ParentEntity {
```

Comment:
Confirm that this class is needed. Its only used in one method which is not used.